### PR TITLE
[ENH] ErrorReport: Pip Freeze, Addons Info, Multiple DSNs

### PR DESCRIPTION
**Pull request by nikicc**
The PR includes the following enhancements for error sending to Sentry:
* Send a list of installed packages (aka pip freeze).
* Send a link to the Orange's schema if provided.
* For addons replace `release` with addon version and store orange version as `orange_version` tag.
* Support sending addon reports to different Sentry projects (different DSNs). If we cannot decide between addon vs core DSN from the stacktrace report is sent to both and marked with a tag `duplicated_in_addon`/`duplicated_in_core`.
* Add logging on production.

### CODE REVIEW

File: orange_web/settings.py [hunk 1]
```diff
@@ -119,4 +119,5 @@
 
 # Error report settings
 ERROR_REPORT_DIR = os.path.abspath("./error_report/")
-ERROR_REPORT_SENTRY_DSN = ''
+# Biolab's testing Sentry project
+ERROR_REPORT_SENTRY_DSN = 'https://c552df1b36cb4334a98ee4c37140af34:d5c900a3f3684d45b0ecc4aaeba540f1@sentry.io/127130'
```
Comment (kernc) [inline orange_web/settings.py]: Is this a secret?

Comment (astaric) [inline orange_web/settings.py]: Nah, this is for a testing project, and can be easily revoked. DSNs for production projects are in a separate (private) file (see settings_production.py)

File: error_report/sentry.py [hunk 2]
```diff
@@ -12,17 +14,18 @@
     "Python34.lib",
     "anaconda3.lib",
     "lib.python3.4",
-    "orange3"
+    "orange3",
+    "orange3-text",
```
Comment (astaric) [inline error_report/sentry.py]: Could addons be recognised by the presence of "orangecontrib" in path?

File: error_report/sentry.py [hunk 3]
```diff
@@ -96,22 +148,31 @@ def create_sentry_report(report):
...
-        logger.exception(ex)
+    def send(sentry_dsn, sentry_report):
+        try:
+            client = Client(sentry_dsn)
+            client.send(**sentry_report)
+        except Exception as ex:
+            # There is nothing we can do if sentry is not available
+            logger.exception(ex)
+
+    # send reports
```
Comment (astaric) [inline error_report/sentry.py]: I would refactor this into something like:

convert to sentry format,
call a function that generates pairs (DSN, report), and then send those reports

File: error_report/sentry.py (hunk 2 above)
Comment (nikicc) [inline error_report/sentry.py]: Refactored to `orangecontrib` instead. Should work for all add-ons.

File: error_report/sentry.py (hunk 3 above)
Comment (nikicc) [inline error_report/sentry.py]: Refactored, please see revised.

File: orange_web/settings_production.py [hunk 4]
```diff
@@ -7,6 +7,7 @@
 DOWNLOAD_DIR = '/srv/download'
 WIDGET_CATALOG = '/srv/chroot_rsync/orange3doc/visual-programming/widgets.json'
 ERROR_REPORT_DIR = '/srv/error_report/'
+LOGGING_DIR = '/var/log/'
```
Comment (nikicc) [inline orange_web/settings_production.py]: Is this the right location?

**Pull request merged**